### ISSUE TITLE
Trouble using where-clause

### ISSUE THREAD

**Issue by kungknut** [donjajo/php-jsondb#14]
Hi!

Thank you for this amazingly easy to use library!

I'm having troubles using the where clause with multiple conditions.

An example entry of my database looks like this;
```json
{
        "id": "fcad9c40-8714-42e0-b579-f002830efb27",
        "title": null,
        "content": "A comment to test this out",
        "author": "Arnold Schwarzenegger",
        "plant_id": "some-plant",
        "created_at": 1579943888,
        "updated_at": null,
        "deleted_at": null
}
```

Now I only want to retrieve the entries where "deleted_at" is set to null using this snippet;

```php
$comments = $jsondb->select('*')
        ->from('comments.json')
        ->where(['plant_id' => $plant_id, 'deleted_at' => null], 'AND')
        ->order_by('created_at', Jajo\JSONDB::DESC)
        ->get();
```

I have also tried setting "deleted_at" to an empty string and compare to that, but this does not seem to work either. No matter what, all results are returned, and no care is taken to whether the "deleted_at" property is set or not.

Out of curiosity I also tried the "OR" option in the exact example above, which results in the comments where "deleted_at" is set to null is returned twice.

Best regards

**Comment by Dictor** [comment]
Hello kungknut!

I wrote some tests for reproducing your issue situation which "WHERE"'s condition is null in [here](https://github.com/Dictor/php-jsondb/commit/e0ca11798732f683772edb5d28a77f8cfed074ac). In my code's test, there is no issue like you and it is working ideal. Please check my test at above link for checking does my code faithfully express your situation.

In addition, "returned twice" issue which you mentioned is fixed in My PR (#15). If you are interested in, check that PR too.

**Comment by donjajo** [comment]
Hi!

Apologies for the delayed response. Was aware from internet over the weekend.
Thanks for reporting this, it has been fixed here https://github.com/donjajo/php-jsondb/commit/9fad45e76e37627cee28a2508b4e99461e15971f 

Please @Dictor, reshare your tests. Not a GitHub pro, I deleted it by mistake after I found the issue and better way to resolve it 

Regards
James
